ui/src/features/WorkspaceSettings/components/MembersSettings.tsx

```diff
@@ -39,14 +39,14 @@ const MembersSettings: React.FC = () => {
   ];
 
   const members = currentWorkspace?.members?.filter(
-    m => "userId" in m && (currentFilter === "all" || m.role === currentFilter),
-  ) as UserMember[];
+    (m): m is UserMember => "userId" in m && (currentFilter === "all" || m.role === currentFilter),
```

Comment:
I think this change is misleading. m is not only UserMember, but could be IntegrationMember too. We do the filter to get an array of UserMembers.

Reply:
I know, it's better than the previous implementation at the very least which was `as UserMember[]` , I did try to `type narrow` on array to get just the `UserMember` and filter out the `IntegrationMember` but this was the best I could come up with. 
 `"userId" in m` states it's a `UserMember` but typescript doesn't get it until i add `is UserMember`. 
 
 Let me know if I can rewrite this in a better way.

Reply:
I think the way it was is the best we can do (there might be a cleverer, more advanced TS way to do it, but I do believe this is a common issue when you make something a type that is two different possible types).
So please revert this change